Cargo.lock

```diff
@@ -245,7 +245,7 @@ dependencies = [
  "num_cpus 1.10.1 (registry+https://github.com/rust-lang/crates.io-index)",
  "rand 0.6.5 (registry+https://github.com/rust-lang/crates.io-index)",
  "rand_xorshift 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
- "serde 1.0.89 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Is this Cargo.lock update needed?

Reply:
Cargo is making the change automatically when I build the project. Pretty sure I could remove the change if you don't want to mix the content of the commit.

Reply:
I removed the change for now and pushed a revised version